controllers/ibmpowervsmachine_controller.go

```diff
@@ -198,7 +198,22 @@ func (r *IBMPowerVSMachineReconciler) getOrCreate(scope *scope.PowerVSMachineSco
 	return instance, err
 }
 
+// handleLoadBalancerPoolMemberConfiguration handles loadbalancer pool member creation flow.
+func (r *IBMPowerVSMachineReconciler) handleLoadBalancerPoolMemberConfiguration(machineScope *scope.PowerVSMachineScope) (ctrl.Result, error) {
+	poolMember, err := machineScope.CreateVPCLoadBalancerPoolMember()
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed CreateVPCLoadBalancerPoolMember %s: %w", machineScope.IBMPowerVSMachine.Name, err)
+	}
+	if poolMember != nil && *poolMember.ProvisioningStatus != string(infrav1beta2.VPCLoadBalancerStateActive) {
+		return ctrl.Result{RequeueAfter: 1 * time.Minute}, nil
+	}
+
+	return ctrl.Result{}, nil
+}
+
 func (r *IBMPowerVSMachineReconciler) reconcileNormal(machineScope *scope.PowerVSMachineScope) (ctrl.Result, error) {
```

Comment:
We should create another issue for refactoring these functions to accept the context from caller,  like `reconcileNormal(ctx context.Context, machineScope *scope.PowerVSMachineScope)`
We are not fully utilizing the contextual logging, May be this can be the first step.

Reply:
Thank you.